spec/services/document_converter_service_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe DocumentConverterService do
+  subject(:convert_document) { described_class.new(document, new_document).call }
+
+  let(:document) { message.attachment }
```

Comment:
within the context of the `app` the use of the term `message` here could be confusing. 

While this service is isolated such that it could be applied to any `ActiveStorage` "backed" object i think it would be clearer to use either a `document` object and affiliated factory OR a mocked object.

other generic terms that could be used might include `storage_record`,`doc_record`, `doc`,  `evidence`, `evidence_rec`